### PR TITLE
adjust alignment of SearchButton

### PR DESCRIPTION
**Pull request by BaderChouaib**

### CODE REVIEW

**Comment by BaderChouaib** [comment]
Yes, it has?
grafik
Are the changes present in your local code base?

**Comment by FiedlerJan** [comment]
> Yes, it has?

Even in your image it is not aligned. The difference to my image is the screen size.

grafik

**Comment by BaderChouaib** [comment]
Sure, fully aligned it now

**Comment by jschurse** [comment]
You are adding padding to the element.
Please align it by setting the text-align property of the parent.
https://developer.mozilla.org/de/docs/Web/CSS/text-align

**Comment by jschurse** [comment]
Alternativly you can ditch the cols on the location serach and use flexbox's flex-grow option
https://developer.mozilla.org/de/docs/Web/CSS/flex-grow

File: app/src/views/Home.vue [hunk 1]
```diff
@@ -54,7 +54,7 @@
                     <v-col cols="8" sm="9" md="4" class="pl-10">
                       <radius />
                     </v-col>
-                    <v-col cols="4" sm="3" md="2" class="pl-10 py-1">
+                    <v-col cols="4" sm="3" md="2" class="pl-12 py-1">
```
Comment (FiedlerJan) [inline app/src/views/Home.vue]: On small screens this works fine.
grafik
But on larger screens it does not work.
grafik

I think the pl class is not the best practise.
@jschurse Do you maybe know a better practise?

Comment (BaderChouaib) [reply app/src/views/Home.vue]: He suggested one above. Trying it out since ive pretty much ran out of options

**Comment by BaderChouaib** [comment]
Used flex-row-reverse on the searchbtn instead of flexgrow on the row since that auto-sized the content to not align with the pictures beneath. Button is still not perfectly aligned to the end of the searchbar

**Comment by FiedlerJan** [comment]
Is there a way to get an equal spacing between all three item (location search bar, radius bar and search button) which remains consistent across all screen sizes?

**Comment by BaderChouaib** [comment]
Flexgrow is really close however i still need padding to ensure aligning. The current state seems promising in terms of responsiveness

**Comment by FiedlerJan** [comment]
> The current state seems promising in terms of responsiveness

The responsiveness looks okay, but let's go into the details. The location searchbar should be larger or at least equal to the radius.

**Comment by BaderChouaib** [comment]
Agreed

**Comment by jschurse** [comment]
Nice!

**Pull request merged**